server/contracts/test/BlockChat.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("assert");
+
+
+const BlockChat = artifacts.require("BlockChat");
+
+contract("BlockChat", accounts => {
+    let token
+
+    before(async () => {
+      token = await BlockChat.deployed()
+    })
+  it("should be deployed", () =>{
+    const address = token.address
+    assert.notEqual(address, 0x0)
+    assert.notEqual(address, '')
```

Comment:
You should be able to write `assert.notEqual('')`, `assert.notEqual(null)` and `assert.notEqual(undefined)`with only one `assert.ok(address) ` as this will check for a truthy value of the string, meaning no empty string, no null and no undefined.